pkg/security/ebpf/c/include/helpers/rate_limiter.h

```diff
@@ -62,17 +56,14 @@ __attribute__((always_inline)) u8 rate_limiter_allow(u32 pid, u64 now, u8 should
 }
 #define rate_limiter_allow_simple() rate_limiter_allow(0, 0, 1)
 
-__attribute__((always_inline)) u8 activity_dump_rate_limiter_allow(u32 rate, u64 cookie, u64 now, u8 should_count) {
+__attribute__((always_inline)) u8 activity_dump_rate_limiter_allow(u16 rate, u64 cookie, u64 now, u16 should_count) {
```

Comment:
If we change the given rate from u32 to u16, which is coming from userspace config, we should also update the the config field, and even enforce the config parameter with some guards